mlflow/server/js/src/shared/web-shared/genai-traces-table/components/filters/TableFilterItem.tsx

```diff
@@ -119,14 +121,18 @@ export const TableFilterItem = ({
     );
 
     // Add individual span filter options
-    result.push(
-      { value: SPAN_CONTENT_COLUMN_ID, renderValue: () => 'Span content' },
-      { value: SPAN_NAME_COLUMN_ID, renderValue: () => 'Span name' },
-      { value: SPAN_TYPE_COLUMN_ID, renderValue: () => 'Span type' },
-    );
+    if (usesV4APIs) {
+      result.push(
+        // TODO: Added via UI sync, but doesn't work in databricks yet. Uncomment
+        // these when the search API supports them
+        // { value: SPAN_CONTENT_COLUMN_ID, renderValue: () => 'Span content' },
+        { value: SPAN_NAME_COLUMN_ID, renderValue: () => 'Span name' },
+        // { value: SPAN_TYPE_COLUMN_ID, renderValue: () => 'Span type' },
```

Comment:
damn sorry can you uncomment these, i forgot to edge guard the comment blocks upstream
```suggestion
        // these when the search API supports them
        { value: SPAN_CONTENT_COLUMN_ID, renderValue: () => 'Span content' },
        { value: SPAN_NAME_COLUMN_ID, renderValue: () => 'Span name' },
        { value: SPAN_TYPE_COLUMN_ID, renderValue: () => 'Span type' },
```